common/culture/traditions/00_regional_traditions.txt

```diff
@@ -426,7 +442,14 @@ tradition_caravaneers = {
 	}
 	
 	is_shown = {
-		always = no
+		# Warcraft
+		any_culture_county = {
+			any_county_province = {
```

Comment:
Too many `any` cycles. need replacing like in the comment above.

Reply:
Adjusted with e3f38ac98 and 6cb3520b2